javascript-practice/src/styles/sections/shopping-cart.css

```diff
@@ -3,13 +3,7 @@
 	margin-bottom: 1.25rem;
 }
 
-.cart-line {
-	margin-bottom: 1rem;
-	height: 1px;
-	background: #EBE8F4;
-}
-
-.thead{
+.thead {
```

Comment:
What does it mean?

Reply:
It means table header. I've fixed it to be more clear